Source/Particles/Pusher/UpdateMomentumBorisWithRadiationReaction.H

```diff
@@ -0,0 +1,82 @@
+
+#ifndef WARPX_PARTICLES_PUSHER_UPDATEMOMENTUM_BORIS_WITHRR_H_
+#define WARPX_PARTICLES_PUSHER_UPDATEMOMENTUM_BORIS_WITHRR_H_
+
+#include <AMReX_REAL.H>
+#include <UpdateMomentumBoris.H>
+
+/* \brief Push the particle's positions over one timestep,
+ *    given the value of its momenta `ux`, `uy`, `uz`.
+ *    Includes Radiation Reaction according to
+ *    https://doi.org/10.1088/1367-2630/12/12/123005 */
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void UpdateMomentumBorisWithRadiationReaction(
+    amrex::Real& ux, amrex::Real& uy, amrex::Real& uz,
+    const amrex::Real Ex, const amrex::Real Ey, const amrex::Real Ez,
```

Comment:
A new type `ParticleReal` was recently added into AMReX, and it is used in WarpX for more flexibility. Could you change the type of relevant arguments from `amrex::Real` to `amrex::ParticleReal`? You may use `UpdateMomentumBoris` as a example.

Reply:
Done.